gap/Magic.gd

```diff
@@ -183,7 +183,7 @@
 #!             Replaces the standard header from &GAPDoc; completely with the header in this LaTeX file.
 #!             Please be careful here, and look at GAPDoc's latexheader.tex file for an example.
 #!         </Item>
-#!         <Mark><A>gapdoc_latex_options</A></Mark>
+#!         <Mark><A>GAPDocLaTeXOptions</A></Mark>
```

Comment:
This needs to be moved from the `scaffold` to the `gapdoc` section, at least if it is supposed to fit with the rest of this PR.